templates/repo/issue/view_content/delete_time.tmpl

```diff
@@ -0,0 +1,14 @@
+{{ if .comment.Time }} <!-- compatibility with pre v1.14 -->
```

Comment:
A comment like this will be visible in the output which is unnecessary. If you really want to do them, do `{{/* a comment */}}`.

Thought in this case, I don't really get what this comment is about because there is not other branch in this condition so it can't be about compatibility, right?

Reply:
Ok, will change that.
As mentioned above, it's about compatibility with comments made before this change. The fallback version is to not show the button, as the comment has no idea about the related `TrackedTime`.

Reply:
That sounds like you need a migration to update all comments to the latest definition so you don't even have to handle that case.

Reply:
As I wrote in the PR title:
> This could probably be resolved once via a migration that matches comments & times with common fields, but that's far out of my comfort zone

Reply:
@noerw can you give me write access?

Reply:
@noerw done